tripy/tests/performance/test_perf.py

```diff
@@ -125,7 +125,7 @@ def func({arg_str}):
 
         return (end - start) / (iterations * 1000.0)
 
-    assert measure_overhead(1) < 60.0
+    assert measure_overhead(1) < 65.0
```

Comment:
Is this still needed? Ideally the inference overhead is unchanged

Reply:
I'll double-check.

Reply:
Works for me locally with the original value, so I hope it will hold up that way in the CI.